web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -197,6 +199,56 @@ fixmystreet.assets.brent.not_found = function(layer) {
     fixmystreet.message_controller.road_found(layer);
 };
 
+fixmystreet.assets.brent.no_asset_message_defaults = {
+    'Parks_and_Open_Spaces' : '#js-brent-park-services',
+    'Highways' : '#js-brent-road-services'
+};
+
+var brent_parkRoadIsFound = {
+    park: false,
+    road: false,
+    isParkorRoad: function() { return this.park || this.road; }
+};
+
+fixmystreet.assets.brent.found_filter = function(layer) {
+    if (fixmystreet.reporting.selectedCategory().category === 'Grass verges / shrub beds - littering') {
+        if (layer.fixmystreet.http_options.params.TYPENAME === 'Parks_and_Open_Spaces') {
+            brent_parkRoadIsFound.park = true;
+        }
+        if (layer.fixmystreet.http_options.params.TYPENAME === 'Highways') {
+            brent_parkRoadIsFound.road = true;
+        }
+        fixmystreet.message_controller.road_found(layer);
+    } else {
+        fixmystreet.message_controller.road_found(layer);
+    }
```

Comment:
Perhaps, given it's always going to be called:
```suggestion
    }
    fixmystreet.message_controller.road_found(layer);
```